### PR TITLE
Fix skip MIP-DNA (for MAF) for tumours wgs for fastq delivery

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
## Description
*This PR fixes the isseu that we got MIP-DNA (for MAF) for tumours wgs for fastq delivery*

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

---

### How to test fix of old samples
- [x] find a case that is wgs application and MIP-DNA for cust000 with one sample and fastq-delivery and tumour
- [x] run the alembic upgrade

### Expected test outcome
- [x] check that that the case was changed to fastq data_analysis

---

### How to test ordering tumour wgs
- [x] order a case with fastq with wgs application, fastq-delivery and is_tumour

### Expected test outcome
- [x] check that that the case got fastq data_analysis

---

### How to test ordering non tumour wgs
- [x] order a case with fastq with wgs application, fastq-delivery and non tumour

### Expected test outcome
- [x] check that that the case got MIP-DNA data_analysis

---

- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by HS
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [x] Document in changelog
- [ ] Deploy this branch
- [ ] Inform to data_analysis, HS, VW

### CODE REVIEW

File: cg/meta/orders/status.py [hunk 1]
```diff
@@ -372,8 +372,8 @@ def store_fastq_samples(
                     raise OrderError(f"Invalid application: {sample['application']}")
                 new_sample.application_version = application_version
                 new_samples.append(new_sample)
-                data_analysis: Pipeline = self.if_type_wgs_do_mip_dna_analysis_for_maf(
-                    application_version
+                data_analysis: Pipeline = self.get_fastq_pipeline(
```
Comment (henrikstranneheim) [inline cg/meta/orders/status.py:375]: What about a mixture of tumour and normals? Will this code handle that or will the pipeline be set for the last sample processed?

Comment (patrikgrenfeldt) [reply cg/meta/orders/status.py:375]: We create on case per fastq sample so it should be fine even in the case of a mixture

**Comment by patrikgrenfeldt** [comment]
Test input:
image

Test result:
image
🆗 tumour -> fastq, non-tumour->mip-dna

**Review by henrikstranneheim — APPROVED** [review]
Well done

**Comment by patrikgrenfeldt** [comment]
Test result migration:
image

in prod (without migration):
image
image
image

same case in stage after migration:
image
🆗 data-analysis was changed from mip-dna to fastq for the wgs tumour sample

**Comment by patrikgrenfeldt** [comment]
deployed on Hasta:
image
image


deployed on clinical-db: 
image

**Pull request merged**